Add script 2 for fine annotation

AntoniaChroni opened the pull request:
<!--Hi there, thanks for your contribution! Please take a moment to fill out this template to facilitate the review of your pull request.-->

<!-- Check all those that apply or remove this section if it is not applicable.-->

# Purpose/implementation Section

## Type of change

- [x] New feature (non-breaking change which adds functionality)


## What scientific question/module of analysis is your analysis addressing?

To add script 2 for fine annotation, i.e., `02-cell-types-annotation-SingleR-fine.Rmd`.

## What GitHub issue does your pull request address?

Closes #112.


## Directions for reviewers 
Tell potential reviewers what kind of feedback you are soliciting.

### Which areas should receive a particularly close look?

Please review for documentation and code logic and clarity.


## Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.

## Documentation Checklist


- [x] This analysis module has a `README` and it is up to date.
- [x] The analytical code is documented and contains comments.

sharonfreshour commented on analyses/cell-types-annotation/02-cell-types-annotation-SingleR-fine.Rmd:
````diff
@@ -0,0 +1,394 @@
+
+```{r, fig.align = "left", results = "asis", message = FALSE, warning = FALSE, echo = FALSE}
+tables1 <- summary_table
+cat("  \n<div align=\"center\", style=\"font-size:80%\">  \n")
````
I think this should be updated to match `pandoc` bug fix from PR #115 

```suggestion
cat("  \n<div align=\"center\" style=\"font-size:80%\">  \n")
```

sharonfreshour commented on analyses/cell-types-annotation/02-cell-types-annotation-SingleR-fine.Rmd:
````diff
@@ -0,0 +1,394 @@
+
+```{r, fig.align = "left", results = "asis", message = FALSE, warning = FALSE, echo = FALSE}
+tables1 <- summary_table
+cat("  \n<div align=\"center\", style=\"font-size:80%\">  \n")
````
I think this should be updated to match `pandoc` bug fix from PR #115 

```suggestion
cat("  \n<div align=\"center\" style=\"font-size:80%\">  \n")
```

sharonfreshour commented on analyses/cell-types-annotation/02-cell-types-annotation-SingleR-fine.Rmd:
````diff
@@ -0,0 +1,394 @@
+cat("  \n\\pagebreak  \n")
+```
+
+Low deltas indicate that the assignment is uncertain, which is especially relevant if the cell’s true label does not exist in the reference. This entire process can be visualized ans inspected using the plotScoreDistribution() function, which displays the per-label distribution of the deltas across cells.
````
```suggestion
Low deltas indicate that the assignment is uncertain, which is especially relevant if the cell’s true label does not exist in the reference. This entire process can be visualized and inspected using the plotScoreDistribution() function, which displays the per-label distribution of the deltas across cells.
```

sharonfreshour reviewed (approved):
Code looks good, it is clear and logical.

I made a few suggestions related to table styles and the `pandoc` bug brought up in PR #115 and also corrected one typo in the report text.

AntoniaChroni commented:
> Code looks good, it is clear and logical.
> 
> I made a few suggestions related to table styles and the `pandoc` bug brought up in PR #115 and also corrected one typo in the report text.

Thanks! Fixed here 45e14f9f14d91e47e7c1269b264b308263e963a0.
